crates/exex/exex/src/notifications.rs

```diff
@@ -13,101 +13,194 @@ use std::{
 };
 use tokio::sync::mpsc::Receiver;
 
-/// A stream of [`ExExNotification`]s. The stream will emit notifications for all blocks.
-pub struct ExExNotifications<P, E> {
-    node_head: Head,
-    provider: P,
-    executor: E,
-    notifications: Receiver<ExExNotification>,
-    wal_handle: WalHandle,
-}
+/// A stream of [`ExExNotification`]s.
+#[derive(Debug)]
+pub struct ExExNotifications<P, E>(Option<ExExNotificationsInner<P, E>>);
```

Comment:
there's now no way to obtain the regular ExExNotificationsWithoutHead stream, right?

Reply:
yes, no way, they fields and the `new` method are private